chrono/schema.py

```diff
@@ -0,0 +1,15 @@
+import graphene
+
+from user import schema as user_schema, mutations as user_mutations
+
+
+class Query(user_schema.Query,):
+    pass
+
+
+
+class Mutation(user_mutations.Mutation,):
+    pass
+
+
+schema = graphene.Schema(query=Query, mutation=Mutation)
```

Comment:
Just for the sake of pep8. https://www.python.org/dev/peps/pep-0008/#blank-lines